pumps/elasticsearch.go

```diff
@@ -355,12 +370,50 @@ func (e *ElasticsearchPump) Init(config interface{}) error {
 		e.log.Info("Index will have date appended to it in the format ", e.esConf.IndexName, "-YYYY.MM.DD")
 	}
 
+	if e.esConf.UseSSL {
```

Comment:
@mativm02 why this is necessary on `Init` func instead of on `getOperator`?

Reply:
![image](https://user-images.githubusercontent.com/83959431/197746711-cf8cbca5-15bf-4389-a89c-59e501fbe3ca.png)
The requirement is asking to apply this logic in the Init function because if it fails, we would just stop the execution when it starts.